File: Solutions/1887. Reduction Operations to Make the Array Elements Equal/1887. Reduction Operations to Make the Array Elements Equal.java
class Solution {
    public int reductionOperations(int[] nums) {
        int res=0,sz=nums.length;
        Arrays.sort(nums);
        for(int i=sz-1;i>0;--i)
            if(nums[i]!=nums[i-1])
                res+=sz-i;
        return res;
    }
}

File: Solutions/1887. Reduction Operations to Make the Array Elements Equal/1887. Reduction Operations to Make the Array Elements Equal.py
class Solution:
    def reductionOperations(self, nums: List[int]) -> int:
        nums.sort()
        res,sz=0,len(nums)
        for i in range(sz-1,0,-1):
            if nums[i]!=nums[i-1]:
                res+=sz-i
        return res
        